src/ray/protobuf/gcs_service.proto

```diff
@@ -190,12 +190,10 @@ message ReportHeartbeatReply {
 }
 
 message CheckAliveRequest {
-  int32 seq = 1;
```

Comment:
this field has not been used in any substantial way. the seq number that's echo-ed back is never consumed by client.